controllers/data.controller.js

```diff
@@ -1,12 +1,22 @@
-const { fetchTopics } = require("../models/data.model");
+const { fetchTopics, fetchArticles } = require("../models/data.model");
 
 exports.getTopics = (req, res, next) => {
   fetchTopics()
     .then((topics) => {
       res.status(200).send(topics);
     })
     .catch((err) => {
-      console.log(err);
+      next(err);
+    });
+};
+
+exports.getArticles = (req, res, next) => {
```

Comment:
just watch the naming of your functions, this could better be named getArticlesById, as you will probably want get Articles for a future ticket